chia/wallet/vc_wallet/vc_drivers.py

```diff
@@ -73,7 +55,7 @@
 STD_COVENANT_PARENT_MORPHER_HASH: bytes32 = STD_COVENANT_PARENT_MORPHER.get_tree_hash()
 EML_TP_COVENANT_ADAPTER_HASH: bytes32 = EML_TP_COVENANT_ADAPTER.get_tree_hash()
 EXTIGENT_METADATA_LAYER_COVENANT_MORPHER_HASH: bytes32 = EXTIGENT_METADATA_LAYER_COVENANT_MORPHER.get_tree_hash()
-VIRAL_BACKDOOR_HASH: bytes32 = VIRAL_BACKDOOR.get_tree_hash()
+REVOCATION_LAYER_HASH: bytes32 = REVOCATION_LAYER.get_tree_hash()
```

Comment:
these kinds of constants seem inconsistently updated to use the pre-computed value